.github/workflows/ci.yml

```diff
@@ -49,6 +49,13 @@ jobs:
           flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics --exclude .git,__pycache__,docs/source/conf.py,old,build,dist,tests/,jina/hub/
           # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
           flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics --exclude .git,__pycache__,docs/source/conf.py,old,build,dist,tests/,jina/hub/
+      - name: Type checking with mypy
+        run: |
```

Comment:
Please remove this from the `CI`, not desired at the moment

Reply:
should we have a local runner like `Makefile`?

Reply:
We are mentioning `mypy` as a tool to help us get better at `type hinting` but not something we want to integrate in our CI for the moment